examples/gnrc_border_router/Makefile

```diff
@@ -4,6 +4,28 @@ APPLICATION = gnrc_border_router
 # If no BOARD is found in the environment, use this default:
 BOARD ?= samr21-xpro
 
+# Things that'll stay on the build command line for sure
+BOARD = nrf52840dongle
+# Same, but I'd eventually like to change the defaults (but getting the info here in the build system is tricky)
+UPLINK=cdc-ecm
+# Don't know any more why I needed them increased, possibly ULA related; also possibly we don't need that many leases if ULA filtering is in there (but see questions there)
+CFLAGS += -DCONFIG_GNRC_NETIF_IPV6_ADDRS_NUMOF=6 -DCONFIG_DHCPV6_CLIENT_PFX_LEASE_MAX=4
+# Things that really should just work out of the box (maybe they do by now)
+# the "nrf802154" thread, which is the radio ifconfig thread
+CFLAGS += -DCONFIG_GNRC_RPL_DEFAULT_NETIF=7
+
+# Better debugging, and also helpful for reviewing https://github.com/RIOT-OS/RIOT/pull/14623
+USEMODULE += netstats_l2
+USEMODULE += netstats_ipv6
+USEMODULE += netstats_rpl
+
+USEMODULE += netstats_neighbor_etx
+USEMODULE += netstats_neighbor_count
+USEMODULE += netstats_neighbor_rssi
+USEMODULE += netstats_neighbor_lqi
+USEMODULE += netstats_neighbor_tx_time
```

Comment:
What are you using those for?

Reply:
IIRC, these are relics from testing some commits that were prerequisites for running MRHOF.